docs/.vuepress/theme/styles/theme-dark.styl

```
@@ -193,15 +193,27 @@ html.theme-dark {
   .tabs-component {
     border-color $tabsComponentBorderColor
 
-    .tabs-component-tabs {
+    ul.tabs-component-tabs {
       border-bottom-color $tabsComponentBorderColor
       background $tabsComponentBackgroundColor
 
-      a {
-        color $fontColor !important
+      .tabs-component-tab {
+        a {
+          color $fontColor
 
-        &:hover {
-          color $linkFontColor !important
+          &:hover {
+            color $linkFontColor
+          }
+        }
+
+        &.is-disabled {
+          a {
+            color $fontColorDisabled
+          }
+
+          &:hover {
+            color $fontColorDisabled
+          }
         }
```

It seems that extra styling for `hover` isn't needed

```suggestion
        &.is-disabled a {
          color $fontColorDisabled
        }
```

---

Done in https://github.com/handsontable/handsontable/pull/8326/commits/0359d0e6d1b0d613cf93bc505f59af1095fe3faf